[KNI] tekton: master branch is 4.22 now

ffromani opened the pull request:
move tekton to 4.22 for master branch

ffromani committed:
[KNI] tekton: master branch is 4.22 now

Signed-off-by: Francesco Romani <fromani@redhat.com>

ffromani commented:
/cc @shajmakh

shajmakh commented:
/lgtm
/test security
4.22 application is still not created (in progress) but we can merge this

ffromani commented:
> /lgtm /test security 4.22 application is still not created (in progress) but we can merge this

let's wait for this a bit; I'll fix 4.21 meanwhile

ffromani commented:
/retest

shajmakh commented:
/retest

shajmakh commented:
/retest

shajmakh commented:
konflux pipeline failures are due to failed coverity check, we've disabled this check for other containers like ztp/cnf-tests we need to do the same here to unblock the PRs. see https://github.com/openshift-kni/cnf-features-deploy/commit/8e6f5462b8fda26fd402e01c91e3a695a2010367

shajmakh committed:
[KNI] tekton: skip coverity task

The task is failing due to improper setup, add an option to skip it and
mark it true to avoid running the task until the setup is fixed (see
CNF-20503) in order to unblock PRs and builds.

Signed-off-by: Shereen Haj <shajmakh@redhat.com>

shajmakh commented:
> konflux pipeline failures are due to failed coverity check, we've disabled this check for other containers like ztp/cnf-tests we need to do the same here to unblock the PRs. see [openshift-kni/cnf-features-deploy@8e6f546](https://github.com/openshift-kni/cnf-features-deploy/commit/8e6f5462b8fda26fd402e01c91e3a695a2010367)

I couldn't push to this branch directly, just cherry-pick this commit https://github.com/openshift-kni/scheduler-plugins/pull/1204/changes/edb03d18396fa6d2aa40058d916782872c80680b it should make the CI pass (skip the failed check)

ffromani commented:
> > konflux pipeline failures are due to failed coverity check, we've disabled this check for other containers like ztp/cnf-tests we need to do the same here to unblock the PRs. see [openshift-kni/cnf-features-deploy@8e6f546](https://github.com/openshift-kni/cnf-features-deploy/commit/8e6f5462b8fda26fd402e01c91e3a695a2010367)
> 
> I couldn't push to this branch directly, just cherry-pick this commit [edb03d1](https://github.com/openshift-kni/scheduler-plugins/commit/edb03d18396fa6d2aa40058d916782872c80680b) it should make the CI pass (skip the failed check)

done, thanks!

shajmakh commented:
/lgtm
Thanks!
